src/module.c

```diff
@@ -9730,10 +9730,30 @@ int RM_CommandFilterArgDelete(RedisModuleCommandFilterCtx *fctx, int pos)
  * with the allocation calls, since sometimes the underlying allocator
  * will allocate more memory.
  */
-size_t RM_MallocSize(void* ptr){
+size_t RM_MallocSize(void* ptr) {
     return zmalloc_size(ptr);
 }
 
+/* Same as RM_MallocSize, except it works on RedisModuleString pointers.
+ */
+size_t RM_MallocSizeString(RedisModuleString* str) {
+    serverAssert(str->type == OBJ_STRING);
+    return objectComputeSize(NULL, str, 0, 0);
+}
+
+/* Same as RM_MallocSize, except it works on RedisModuleDict pointers.
+ * Note that the returned value is only the overhead of the underlying structures,
+ * it does not include the allocation size of the keys and values.
+ */
+size_t RM_MallocSizeDict(RedisModuleDict* dict) {
+    /* See streamRadixTreeMemoryUsage */
+    size_t size = sizeof(RedisModuleDict);
```

Comment:
i think we're missing the `sizeof(rax)`

Reply:
then i guess we are also missing it in `streamRadixTreeMemoryUsage`, can you verify?

Reply:
yes, you're right, let's fix as part of this PR?